admin/resources/js/components/form/TextArea.tsx

```diff
@@ -28,15 +33,25 @@ const TextArea: React.FunctionComponent<TextAreaProps> = ({
   const error = get(errors, name)?.message;
 
   return (
-    <div>
-      <label htmlFor={id}>{label}</label>
-      <textarea
-        id={id}
-        {...register(name, rules)}
-        aria-invalid={error ? "true" : "false"}
-        {...rest}
-      />
-      {error && <span role="alert">{error}</span>}
+    <div data-h2-margin="b(bottom, xxs)">
+      <InputWrapper
+        inputId={id}
+        label={label}
+        required={!!rules.required}
+        context={context}
+        error={error}
+      >
+        <textarea
+          data-h2-padding="b(all, xxs)"
+          data-h2-radius="b(s)"
+          data-h2-border="b(darkgray, all, solid, s)"
+          style={{ width: "100%", height: "5rem" }}
```

Comment:
Should the `height` be a prop that can control this value (also, maybe min-height instead to allow it to stretch?)?

Reply:
Hmm, ya I think having a prop for the min-height would be better 👍